test/inductor/test_compiled_optimizers.py

```diff
@@ -47,12 +47,16 @@ class KernelCounts(NamedTuple):
 KERNEL_COUNT_OVERRIDES = {
     "test_rmsprop_foreach_weight_decay_cpu": 12,
     "test_nadam_foreach_weight_decay_momentum_decay_cpu": 20,
+    "test_nadam_foreach_cpu": 12,
+    "test_nadam_foreach_momentum_decay_cpu": 12,
+    "test_nadam_foreach_weight_decay_momentum_decay_decoupled_weight_decay_cpu": 12,
     "test_adamw_foreach_amsgrad_capturable_cuda": 3,
     "test_adamw_amsgrad_capturable_cuda": 6,
     "test_adam_amsgrad_capturable_cuda": 6,
     "test_adadelta_foreach_weight_decay_maximize_cpu": 12,
     "test_adadelta_foreach_rho_weight_decay_cpu": 12,
     "test_adadelta_foreach_weight_decay_cpu": 12,
+    "test_asgd_recompile_single": 16,
```

Comment:
NIT: I think this value is defined here

https://github.com/pytorch/pytorch/blob/44632939b5aa0c3e658d807c294df8fb3018fb91/test/inductor/test_compiled_optimizers.py#L320-L323